app/src/main-process/menu/build-default-menu.ts

```diff
@@ -512,6 +522,14 @@ function getPushLabel(
   return __DARWIN__ ? 'Force Push' : 'Force P&ush'
 }
 
+function getStashedChangesLabel(isStashedChangesVisible: boolean): string {
+  if (isStashedChangesVisible) {
+    return __DARWIN__ ? 'Hide Stashed Changes' : 'H&ide stashed'
```

Comment:
Is there a reason we've dropped this last word?

```suggestion
    return __DARWIN__ ? 'Hide Stashed Changes' : 'H&ide stashed changes'
```

Reply:
I would say fat-fingered, but my fingers are built more for poking.